src/vs/workbench/contrib/chat/browser/chatSetup.ts

```diff
@@ -866,7 +866,7 @@ class ChatSetupWelcomeContent extends Disposable {
 		}
 
 		// Limited SKU
-		const limitedSkuHeader = localize({ key: 'limitedSkuHeader', comment: ['{Locked="[]({0})"}'] }, "$(sparkle-filled) We now offer [Copilot for free]({0}) with 2,000 code completions and 50 chat messages per month.", defaultChat.skusDocumentationUrl);
+		const limitedSkuHeader = localize({ key: 'limitedSkuHeader', comment: ['{Locked="[]({0})"}'] }, "$(sparkle-filled) We now offer [Copilot for free]({0}).", defaultChat.skusDocumentationUrl);
```

Comment:
Do you also need to lock the `$(sparkle-filled)` or use the positional args to avoid someone trying to localize it?

Reply:
Umm, I'm not an expert at localization so not positive. This was what we had previously before adding the limits wording. So I was just reverting to that.

Reply:
Thanks! @roblourens suggested string concatenation as a simpler alternative to locking so I've done that.